docs/usage/getting-started/troubleshooting.md

```diff
@@ -0,0 +1,66 @@
+# Troubleshooting Renovate
+
+Learn how to troubleshoot problems with Renovate, where to find the logging output, and how to get help if needed.
+
+## Getting the logs
+
+Renovate logs the most recent jobs.
+Where you can find the logs depends on how you're running Renovate.
+
+### GitHub Hosted app
+
+Each pull request from Renovate contains a link to the WhiteSource Renovate Dashboard in the PR body text.
+
+The text you're looking for is:
+
+> This PR has been generated by WhiteSource Renovate. View repository job log here.
+
+Click on the blue text "here" to go to the WhiteSource Renovate Dashboard.
+Sign in with your GitHub or GitLab account.
```

Comment:
That's because GitLab still doesn't work for us as a hosted platform isn't it?

Could you use your GitLab account to access logs made on GitHub repositories? Stupid idea maybe... 😄

Maybe if you cannot really do anything functional when logged in with GitLab, maybe that option should be removed from the login screen???

Reply:
No, you can't use gitlab to access github, and I need to remove the login option soon

Reply:
> No, you can't use gitlab to access github, and I need to remove the login option soon

Okay! 😉